packages/website/src/pages/navigation/SubNavigation.tsx

```diff
@@ -83,15 +83,15 @@ const code = `
     export default () => {
         const exampleItems = [
             {label: 'Avatar', id: 'avatar'},
-            {label: 'Titanic', id: 'titanic'},
+            {label: <div className="body-m py1">Titanic</div>, id: 'titanic', description: 'Cupcake ipsum dolor sit amet. Lemon drops croissant sesame snaps cookie jelly beans tootsie roll muffin. Liquorice liquorice fruitcake tiramisu sesame snaps sugar plum lollipop gummi bears cookie.'},
```

Comment:
`<div className="body-m py1">Titanic</div>`

This is the format the label should have when using a description? Is this a UX recommendation?

Reply:
it is not a UX recommendation, but since the label allows for a `React.Node`, it was just a simple example since we did not have one

Reply:
I understand what you want to do and it's not stupid. On the other hand, what I would like to see in his examples is also the UX recommendations.

Because if someone from the UX team tells me to add a description, I will obviously copy your example and I risk taking the format of this label thinking that this is what the UX team wants.

Maybe you could add an example with this possibility to add a ReactNode instead of a string? For exemple for the line with `The Avengers`, a different line of your example.

Reply:
(disclaimer I did not read all this thread)

I think it should not be possible to have one tab if a description and the other without? 🤔

Reply:
Also I think this would deserve a new example since this is a variant of the default pattern 😄

Reply:
> > I'm not a fan of having with and without description in the same subnav 👀
> 
> I spoke to @guscoveo about this, I added some demo's to show what should be used when we want to have a description along with a non description.
> 
> I also adjusted the css logic to apply the correct padding as per the Figma mockups.
> 


That's better but then we should enforce this style otherwise it will be forgotten 🙈 

And you should split the example the default one and the withDescriptions one. Poke me on slack if you need help on how to do it 😄

Reply:
Will do! Thanks 🙏